spec/support/requests_spec_helper.rb

```diff
@@ -8,7 +8,7 @@ def api(version = 0.0)
     "/api/v#{version}"
   end
 
-  def disable_tenancy
-    stub_const("ENV", "BYPASS_TENANCY" => "true")
+  def bypass_tenancy(&block)
```

Comment:
@syncrou 

Since tenancy is enabled out of the box why do we have these specs with the around block for bypassing tenancy in every model.

If we have to test that Tenancy filtering works

We create 2 portfolio records one with with Tenant1.ID and another one with Tenant2.ID and then
ActsAsTenant.with_tenant(tenant1)
  We get tenant1 portfolio

The above logic will work for all object types if it works for one object type

If we want to check that BYPASS tenancy works
we have 1 single test that does

with_modified_env(BYPASS_TENANCY: true)
       gets 2 records back